microprofile/cdi/src/main/resources/META-INF/helidon/native-image/weld-proxies.json

```diff
@@ -0,0 +1,21 @@
+[
+    {
+        "bean-class": "org.jboss.weld.event.EventImpl",
+        "ifaces": [
+            "javax.enterprise.event.Event",
+            "org.jboss.weld.events.WeldEvent"
+        ]
+    },
+    {
+        "bean-class": "org.jboss.weld.bean.builtin.InstanceImpl",
+        "ifaces": [
+            "javax.enterprise.inject.Instance"
+        ]
+    },
+    {
+        "bean-class": "org.glassfish.jersey.weld.se.WeldRequestScope",
+        "ifaces": [
+            "org.glassfish.jersey.weld.se.WeldRequestScope"
```

Comment:
These are the same; maybe that's OK; I don't know.

Reply:
This is needed for the current implementation to work (the bean class and interfaces were gathered from runtime).